tyk-pro/templates/bootstrap-post-install.yaml

```
@@ -2,7 +2,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: "bootstrap-post-install-{{ include "tyk-pro.fullname" . }}"
+  name:  {{ .Values.jobs.bootstrapJobName }}
```

The previous job name was `"bootstrap-post-install-{{ include "tyk-pro.fullname" . }}"` but right now, we are just passing `"bootstrap-post-install` part without including full installation name.

Should we update the `metadata.name` as follows:
```suggestion
  name:  {{ .Values.jobs.bootstrapJobName }}-{{ include "tyk-pro.fullname" . }}
```

what do you think?

---

This will induce complications as we are using the job name during the cleanup.